src/app/public/modules/toast/toaster.component.html

```diff
@@ -1,9 +1,13 @@
 <div
   class="sky-toaster"
+  (focusin)="onFocusIn()"
+  (focusout)="onFocusOut()"
+  (mouseenter)="onMouseEnter()"
+  (mouseleave)="onMouseLeave()"
```

Comment:
Maybe not something we're interested in solving today, but was there any discussion on how to halt the timer for users on mobile devices?

Reply:
We probably won't do this for mobile. When we document this, we will state that in order to use the auto-close feature, your application must provide a way to access toast messages that are auto-closed without the user noticing, so worst case is the user misses the toast but will have another element (like a bell icon) with a persistent indicator where the user can go access the toast message.